packages/fast-form-generator-react/src/form/form-item.array.tsx

```diff
@@ -98,10 +121,38 @@ class FormItemArray extends React.Component<IFormItemArrayProps & IManagedClasse
         );
     }
 
+    public componentDidUpdate(): void {
+        this.state.hideOptionMenu
+            ? window.removeEventListener("click", this.handleWindowClick)
+            : window.addEventListener("click", this.handleWindowClick);
```

Comment:
window click event is IE9+, we may want to use document instead of window because of that. It doesn't hurt to be backwards compatible in this case so I would change it.

Reply:
Sure.